lighty-modules/lighty-netconf-sb/src/test/java/io/lighty/modules/southbound/netconf/tests/SampleConfigTest.java

```diff
@@ -45,7 +45,7 @@ public void loadTopLevelClusterModelsFromJsonConfig() throws Exception {
                 .getEffectiveModelContext().getModules().size();
         assertTrue(lightyController.shutdown(TIME_OUT, TimeUnit.SECONDS));
 
-        assertEquals(loadedModulesSize, 17);
+        assertEquals(loadedModulesSize, 15);
```

Comment:
No, you have to adapt revisions in: `sampleConfigCluster.json`.

Reply:
removed this commit and changed back to original size.